packages/at_secondary_server/test/test_utils.dart

```diff
@@ -96,7 +96,7 @@ late Function(dynamic data) socketOnDataFn;
 // ignore: unused_local_variable
 late Function() socketOnDoneFn;
 // ignore: unused_local_variable
-late Function(Exception e) socketOnErrorFn;
+late Function(Exception e, StackTrace st) socketOnErrorFn;
```

Comment:
Prevent irrelevant but annoying exception messages in various places in our test harnesses